system-variables.md

```diff
@@ -1325,9 +1325,9 @@ This variable is associated with a feature available since v6.2.0 and not usable
 
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
-- Default value: In v6.1.0, the default value is `ON`. After v6.1.0, the default value is `OFF`.
+- Default value: In v6.1.0, the default value is `ON`.
 - Since v6.1.0, the [Join Reorder](/join-reorder.md) algorithm of TiDB supports Outer Join. This variable controls the support behavior. The default value is `OFF`, which means the Join Reorder's support for Outer Join is disabled by default.
-- For a cluster upgraded from a version earlier than v6.1.0, the default value is `OFF`. For a cluster upgraded from v6.1.0, the default value is `ON`.
+- If the TiDB version before the upgrade is earlier than v6.1.0, the default value of this variable after the upgrade is `ON`.
```

Comment:
The above line reuses the translation in the dev version: https://docs.pingcap.com/tidb/dev/system-variables#tidb_enable_outer_join_reorder-new-in-v610